modules/virt-latency-checkup-web-console.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-running-cluster-checkups.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-latency-checkup-web-console_{context}"]
+= Running a latency checkup in the web console
+
+Run a latency checkup to verify network connectivity and measure the latency between two virtual machines attached to a secondary network interface.
+
+.Prerequisites
+
+* You must add a `NetworkAttachmentDefinition` to the namespace.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *Checkups* in the web console.
+. Click the *Network latency* tab.
+. Click *Install permissions*.
+. Click *Run checkup*.
+. Enter a *Name* for the checkup.
+. Select a *NetworkAttachmentDefinition* from the dropdown menu.
+. Optional: Set the *Sample duration (seconds)*.
+. Optional: Enable *Set maximum desired latency (milliseconds)*, then define the time interval.
+. Optional: Enable *Select nodes* to target specific nodes, then select the *Source node* and *Target node*.
```

Comment:
Same concern as above. Do we need bold here? Or
```suggestion
. Optional: Set a dureation for (blah) in the *Sample duration (seconds)* field.
. Optional: Enable latency by selecting *Set maximum desired latency (milliseconds)* and defining the time interval.
. Optional: Target specific nodes by selecting *Select nodes* and specifing the *Source node* and *Target node*.
```
I don't have access to the console, so forgive me if this is incorrect.